src/Rings/mpoly-graded.jl

```diff
@@ -754,7 +754,17 @@ function addeq!(a::MPolyDecRingElem, b::MPolyDecRingElem)
 end
 
 length(a::MPolyDecRingElem) = length(forget_decoration(a))
+
+@doc raw"""
+    total_degree(f::MPolyDecRingElem)
+
+Return the total degree of `f`.
+
+!!! note
+    Given a set of variables ``x = \{x_1, \ldots, x_n\}``, the *total degree* of a monomial ``x^\alpha=x_1^{\alpha_1}\cdots x_n^{\alpha_n}\in\text{Mon}_n(x)`` is the sum of the ``\alpha_i``. The *total degree* of a polynomial `f`  is the maximum of the total degrees of its monomials. In particular, the notion of total degree does not dependent on weights given to the variables.
```

Comment:
```suggestion

    Given a set of variables ``x = \{x_1, \ldots, x_n\}``, the *total degree* of a monomial ``x^\alpha=x_1^{\alpha_1}\cdots x_n^{\alpha_n}\in\text{Mon}_n(x)`` is the sum of the ``\alpha_i``. The *total degree* of a polynomial `f`  is the maximum of the total degrees of its monomials. 

!!! note
    The notion of total degree does not dependent on weights given to the variables.
```
Would this be an option too? Then the most dangerous part is highlighted only.